### PR TITLE
Add release notes for version 1.0.0.0-beta1

### PR DESCRIPTION
**Pull request by ohltyler**
Signed-off-by: Tyler Ohlsen <ohltyler@amazon.com>

### Description
Adds release notes for 1.0.0.0-beta1
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Version 1.0.0.0-beta1 Release Notes
+
+Compatible with OpenSearch 1.0.0
+
+### Infrastructure
+
+* Fix: use AWS_SECRET_ACCESS_KEY. ([#3](https://github.com/opensearch-project/anomaly-detection/pull/3))
```
Comment (dblock) [inline release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md:7]: This one was a transient thing during the rename, remove.

File: release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* Delete ad-cli ([#5](https://github.com/opensearch-project/anomaly-detection/pull/5))
+* Fix access key typo, added VSCode gitignores ([#8](https://github.com/opensearch-project/anomaly-detection/pull/8))
```
Comment (dblock) [inline release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md:9]: Same, all the 3 below are not useful in release notes, just things we introduced then fixed.

File: release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+* Fixed integration tests. ([#18](https://github.com/opensearch-project/anomaly-detection/pull/18))
+
+### Documentation
+
+* Removing few lingering Elasticsearch strings in the repo ([#6](https://github.com/opensearch-project/anomaly-detection/pull/6))
+* Updating the license and docs ([#7](https://github.com/opensearch-project/anomaly-detection/pull/7))
+* Add SPDX license header to all files ([#10](https://github.com/opensearch-project/anomaly-detection/pull/10))
+* Updating the Notice.txt to relfect the right software being used ([#11](https://github.com/opensearch-project/anomaly-detection/pull/11))
+* Updated copyright and external links. ([#12](https://github.com/opensearch-project/anomaly-detection/pull/12))
+* Add release notes for version 1.0.0.0-beta1 ([#24](https://github.com/opensearch-project/anomaly-detection/pull/24))
```
Comment (dblock) [inline release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md:20]: This is a bit meta, let's remove?

**Review by dblock — CHANGES_REQUESTED** [review]
I propose we remove anything that's not actually meaningful to the release?

**Comment by ohltyler** [comment]
> I propose we remove anything that's not actually meaningful to the release?

Guidance before has been to include all commits, regardless of how meaningful they are, to the release notes. We may bring this discussion up again if we want to filter out certain commits.

**Comment by dblock** [comment]
> > I propose we remove anything that's not actually meaningful to the release?
> 
> Guidance before has been to include all commits, regardless of how meaningful they are, to the release notes. We may bring this discussion up again if we want to filter out certain commits.

I'll approve for now, but to me that makes little sense for a 1.0 release.

**Review by yizheliu-amazon — APPROVED** [review]

**Review by ylwu-amzn — APPROVED** [review]

File: release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md (hunk 2 above)
Comment (ylwu-amzn) [reply release-notes/opensearch.anomaly-detection.release-notes-1.0.0.0-beta1.md:9]: Do we have any guideline for what should be included in release notes in OpenSearch release? I remember we include all commits in ODFE release.

**Pull request merged**